### DISCUSSION TITLE
Charged after cancelling my subscription

### DISCUSSION THREAD

**Discussion by raiadi2008** [vercel/community#4330] [Account Issue]
### Question

Hi, I cancelled my pro subscription and deleted my team but still, my card was charged for a subscription after doing that. How Can I get refund?

**Comment by amyegan** [comment]
Sorry you're having issues with canceled subscription charges. You can use the [support form](https://vercel.com/help#issues) to submit a case to the Customer Success Team under the `Billing` problem area. That will route your case to the people who can help with this 🙇

**Reply by raiadi2008** [reply]
Hi @amyegan I have already done that. it's been 4 days and I haven't received a reply so far.

**Reply by amyegan** [reply]
If you let me know the case number, I can check the status for you

**Reply by raiadi2008** [reply]
00162356. This is my case number. I have also attached my bill in the same. @amyegan
